crates/sui-graphql-rpc/src/types/object.rs

```diff
@@ -76,6 +68,34 @@ pub(crate) struct ObjectKey {
     version: u64,
 }
 
+#[derive(Union, Clone)]
+pub enum ObjectOwnerNew {
```

Comment:
suggestion: I would rename the ObjectOwner in owner.rs to IOwner to 1) match draft_schema 2) free up the namespace for ObjectOwner, and then extend the IOwner enum to include: Owner, Address, Object, Coin, StakedSui, CoinMetadata, MoveObject, Movepackage, although I think the second step can be done separately?

Reply:
+1 to the first suggestion to get you unblocked (rename the current `ObjectOwner` in `owner.rs` to `IOwner`), but yes, let's leave the the rest of the interface trickery for another day (in particular, I'll get to it with the changes related to upcasts), because otherwise, I fear an absolute dumpster fire of a merge conflict.